.github/workflows/deploy-avax-strategy.yaml

```
@@ -0,0 +1,85 @@
+        description: 'Collateral tokens comma separated'
+        required: false
+        default: '0x'
+      minCollateralRatios:
+        description: 'Minimum collateral ratio comma separated'
+        required: false
+        default: '0.01'
+    
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    environment: 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)